package.json

```diff
@@ -2,9 +2,18 @@
   "name": "analysis-ui-components",
   "version": "0.18.0",
   "description": "React components to build analysis UI",
-  "main": "lib/components/index.js",
-  "module": "lib-esm/components/index.js",
-  "types": "lib-esm/components/index.d.ts",
+  "exports": {
+    "./app": {
+      "types": "./lib-esm/app/index.d.ts",
+      "require": "./lib/app/index.js",
+      "default": "./lib-esm/app/index.js"
+    },
+    "./components": {
+      "types": "./lib-esm/components/index.d.ts",
+      "require": "./lib/components/index.js",
+      "default": "./lib-esm/components/index.js"
+    }
+  },
```

Comment:
@stropitek We need to find better names for top-level directories/exports.
The existing separation is confusing:
- `./app` contains components
- `./components` contains things that are not components (hooks for example)

Reply:
could `./app` be just `./demo` ?

Reply:
No, IMO the name doesn't make sense, and I just created the demo directory in this PR to separate the public app API from the demo app built from this repo.